internal/topic/topicwriterinternal/message.go

```diff
@@ -157,27 +146,26 @@ func (m *messageWithDataContent) readDataToTargetCodec(codec rawtopiccommon.Code
 	m.bufCodec = codec
 	m.bufEncoded.Reset()
 
-	encoder, err := m.encoders.CreateLazyEncodeWriter(codec, &m.bufEncoded)
-	if err != nil {
-		return err
-	}
-
 	reader := m.Data
 	if reader == nil {
 		reader = &bytes.Reader{}
 	}
-	bytesCount, err := io.Copy(encoder, reader)
-	if err == nil {
-		err = encoder.Close()
+
+	buf := &bytes.Buffer{}
```

Comment:
What about use [xsync.Pool](https://pkg.go.dev/github.com/ydb-platform/ydb-go-sdk/v3/internal/xsync#Pool) - internal type wrapper over sync pool for get bytes buffer?

It is often operation - it will need for every message.